src/test/java/pl/solvd/carina/WebTest.java

```diff
@@ -0,0 +1,85 @@
+package pl.solvd.carina;
+
+import com.qaprosoft.carina.core.foundation.IAbstractTest;
+import com.zebrunner.agent.core.annotation.TestLabel;
+import com.zebrunner.carina.core.registrar.ownership.MethodOwner;
+import com.zebrunner.carina.core.registrar.tag.Priority;
+import com.zebrunner.carina.core.registrar.tag.TestPriority;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.interactions.Actions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class WebTest implements IAbstractTest {
```

Comment:
Not really good name
```suggestion
public class CarinaDemoHeaderTest implements IAbstractTest {
```

Reply:
done